android/src/main/kotlin/io/didomi/fluttersdk/DidomiEventStreamHandler.kt

```diff
@@ -0,0 +1,38 @@
+package io.didomi.fluttersdk
+
+import io.didomi.sdk.events.EventListener
+import io.didomi.sdk.events.HideNoticeEvent
+import io.didomi.sdk.events.ReadyEvent
+import io.didomi.sdk.events.ShowNoticeEvent
+import io.flutter.plugin.common.EventChannel
+
+/**
+ * Handler for SDK events
+ */
+class DidomiEventStreamHandler: EventChannel.StreamHandler, EventListener() {
+    private var eventSink: EventChannel.EventSink? = null
+
+    override fun onListen(arguments: Any?, events: EventChannel.EventSink?) {
+        this.eventSink = events
+    }
+
+    override fun onCancel(arguments: Any?) {
+        this.eventSink = null
+    }
+
+    override fun showNotice(event: ShowNoticeEvent) {
+        sendEvent("onShowNotice")
```

Comment:
would it make sense to have method/event names as constants somewhere?

Reply:
As far as I understand, it is not possible to share the constant value between the different platforms. As these values will be used only once per platform, there is not much interest in adding a Constant for them imo.